### DISCUSSION TITLE
How Long Does It Take for a Branch Committed in Development to Move to Stable?

### DISCUSSION THREAD

**Discussion by pmenotti** [mpdf/mpdf#2027] [General]
I'm currently working on a project that requires a bug fix that I identified and corrected in the development branch of the mpdf repository, specifically authored by  @finwe. Can anyone provide insights on how long it typically takes for changes in the development branch to be reviewed, merged, and released in the stable version? Any information on the usual release cycle or process would be highly appreciated.

**Comment by finwe** [comment]
Usual release cycle is _as I see fit_, once a significant amount of fixes has accumulated.
